### PR TITLE
Add ExCoGi xPath scraper

### PR DESCRIPTION
**Pull request by halorrr**
Of note, there are scene dates on the site but not on the scene page. I could not figure out a subscraper method to grab them.

### CODE REVIEW

File: scrapers/ExCoGi.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      Title: $info/h4/strong/text()
+      Details: //article[@id='about']/p/text()
+      Image:
+        selector: //img[@class='img-responsive center-block main-preview']/@src
+        postProcess:
+          - replace:
+            - regex: ^
+              with: https://exploitedcollegegirls.com
+      Studio:
+        Name: Exploited College Girls
```
Comment (bnkai) [inline scrapers/ExCoGi.yml:21]: forgot the `fixed`
``yaml
      Studio:
        Name: 
          fixed: Exploited College Girl
```

File: scrapers/ExCoGi.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    url:
+      - exploitedcollegegirls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class='info']
+    scene:
+      Title: $info/h4/strong/text()
+      Details: //article[@id='about']/p/text()
```
Comment (bnkai) [inline scrapers/ExCoGi.yml:13]: some scenes have a lot of paragraphs eg `https://exploitedcollegegirls.com/free/scene/487/1339/ecg-video-taylor`
```yaml
      Details:
        selector: //article[@id='about']/p/text()
        concat: "\n"
```

File: scrapers/ExCoGi.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        selector: //img[@class='img-responsive center-block main-preview']/@src
+        postProcess:
+          - replace:
+            - regex: ^
+              with: https://exploitedcollegegirls.com
+      Studio:
+        Name: Exploited College Girls
+      Tags:
+        Name: $info//a/text()
+      Performers:
```
Comment (bnkai) [inline scrapers/ExCoGi.yml:24]: In all the scenes i tried it seems that 
```yaml
      Performers:
        Name: //meta[@name="twitter:description"]/@content
```
is enough. Can you verify with your samples to make sure it works?

Comment (halorrr) [reply scrapers/ExCoGi.yml:27]: Good find! This works with all the sample scenes I had!

**Comment by bnkai** [comment]
Couldn't find a way for the dates either. We take what we can....
Merging

**Pull request merged**